android_jni/avifandroidjni/src/main/jni/CMakeLists.txt

```diff
@@ -0,0 +1,26 @@
+cmake_minimum_required(VERSION 3.7)
+
+project(avif_android_jni)
+
+# Perform a static build of libavif and link it with the jni shared library so
+# that we end up with a single libavif_android.so file in the end.
+#
+# By default, we build libavif using libgav1. To change that behavior, set up
+# the android builds for the desired decoder and change the variables below
+# accordingly.
```

Comment:
This requires building libyuv for android and making it available as part of the CMake android build. So this is not being done as part of this PR. It can be done in a follow-up. This will be cmake-only change and will not change any of the code.